extension/src/popup/views/Account/index.tsx

```diff
@@ -309,7 +309,7 @@ export const Account = () => {
                   />
                 </div>
               )}
-              {!isFunded && !hasError && (
+              {!isFunded && !hasError && !error?.horizon && (
```

Comment:
don't show the unfunded message if we have a horizon message bc we're not sure if you're funded or not